src/cozmo/version.py

```diff
@@ -17,5 +17,5 @@
 # Specify the version of cozmoclad that this package requires
 # Releases of the Cozmo package must specify an exact cozmoclad release
 # to ensure compatibility with a specific release of the ios/android app.
-#__cozmoclad_version__ = None
-__cozmoclad_version__ = "1.0.1"
+__cozmoclad_version__ = None
```

Comment:
99% of the time they shouldn't need to manually install as it will pull the latest published version of the cozmoclad package if they don't already have one installed, and that'll match the latest app in the app store